frontend/src/containers/UpdateCourses/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+import { useSelector } from "react-redux";
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Button,
+  Text,
+  useToast
+} from '@chakra-ui/react';
+
+import { postScrapSchedule } from 'services/api';
+
+import { InputPassword, InputText } from 'components/Forms';
+import { InfoToast, SuccessToast } from 'components/Toast';
+import { Bauhaus } from 'components/Bauhaus';
+import Alert from './Alert';
+import Info from './Info';
+import './styles.css';
+
+const UpdateCourses = ({ history }) => {
+  const isMobile = useSelector((state) => state.appState.isMobile);
+  const toast = useToast();
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors, isSubmitting, isSubmitSuccessful },
+  } = useForm();
+
+  const onSubmit = async (values) => {
+    try {
+      InfoToast(
+        'Sedang memperbaharui jadwal',
+        isMobile
+      );
+      postScrapSchedule(values);
```

Comment:
ini gua merge besok pagi ya kak @nugrahaa878, mau nambahin error msg yg baru ditambah di be dulu